http/src/request/mod.rs

```diff
@@ -5,6 +5,12 @@ pub mod sticker;
 pub mod template;
 pub mod user;
 
+#[deprecated(
+    note = "this will be removed in a future major version",
+    since = "0.7.2"
+)]
```

Comment:
fun fact is that #[deprecated] on modules does nothing on the re-exports:

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=1705f2ee331b5d401ad4ac5f54e2c534